t/test-suite.lisp

```diff
@@ -88,20 +101,29 @@
   (setf *tmpdir* (tmpdir:mkdtemp))
   (setf *sqlite-conn*
         (cl-dbi:connect :sqlite3
-                        :database-name (merge-pathnames (make-pathname :name "cl-migratum" :type "db")
+                        :database-name (merge-pathnames
+                                        (make-pathname :name "cl-migratum" :type "db")
                                                         *tmpdir*)))
   (setf *provider* (cl-migratum.provider.local-path:make-provider (list *migrations-path*)))
   (setf *dbi-driver*
         (migratum.driver.dbi:make-driver *provider* *sqlite-conn*))
-  (setf *rdbms-postgresql-driver*
-        (migratum.driver.rdbms-postgresql:make-driver *provider*
-                                                      `(:host ,(or (uiop:getenv "PGHOST") "localhost")
-                                                        :database ,(or (uiop:getenv "PGDATABASE") "migratum")
-                                                        :user-name ,(or (uiop:getenv "PGUSER") "migratum")
-                                                        :password ,(or (uiop:getenv "PGPASSWORD") "FvbRd5qdeWHNum9p")))))
-
+  (let ((auth
+          `(:host ,(or (uiop:getenv "PGHOST") "localhost")
+            :database ,(or (uiop:getenv "PGDATABASE") "migratum")
+            :user-name ,(or (uiop:getenv "PGUSER") "migratum")
+            :password ,(or (uiop:getenv "PGPASSWORD") "FvbRd5qdeWHNum9p"))))
+    (setf *postmodern-postgresql-driver*
+          (migratum.driver.postmodern-postgresql:make-driver
+           *provider*
+           (list* :port *postmodern-postgresql-port auth))
```

Comment:
@K1D77A , typo here as well -- `*postmodern-postgresql-port` should be `*postmodern-postgresql-port*`.